handbook/community.md

```diff
@@ -236,6 +236,26 @@ IT and security teams love Fleet because of its flexibility and conventions. Ins
 
 That means better privacy. Better device performance. And better data, with less noise.
 
+## Sponsoring Events
```

Comment:
[sentence case](https://fleetdm.com/handbook/brand#how-to-write-headings-subheadings)